.changes/next-release/bugfix-54c0cf8a-9df5-4673-bf16-61a650434abc.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Not show code scan tool window tab when switch to Problem ToolWindow"
```

i do not know how to parse this statement
```suggestion
  "description" : "Do not always show 'Amazon Q Code Issues' tab when switching to the 'Problems' tool window"
```
?